vasyl_varyshchuk/Week_1/task_1/max_of_list.py

```diff
@@ -0,0 +1,28 @@
+'''
+This module finds maximum integer value from list of strings.
+
+'''
+
+data = ['one', '1', 'two', '2', 'ten', '10', '15', '-2', 'some long text goes here', '16.9', '&']
+
+def get_max_value(iterable):
+
+    numbers = []
+
+    for item in iterable:
+
+        try:
+            numbers.append(int(float(item)))
+        except ValueError:
+            print("Could not convert '{}' to an integer.".format(item))
+
+    if len(numbers) != 0:
```

Comment:
Why do you need such complicated conditions?

What about something like this?

```
    return max(numbers) if numbers else None
```

Or something similar.